Mathlib/RingTheory/Ideal/Maps.lean

```diff
@@ -908,3 +908,14 @@ lemma coe_ideal_map (I : Ideal A) :
     Ideal.map f I = Ideal.map (f : A →+* B) I := rfl
 
 end AlgHom
+
+namespace Algebra
+
+variable {R : Type*} [CommSemiring R] (S : Type*) [Semiring S] [Algebra R S]
+
+@[simps!]
+def idealMap (I : Ideal R) : I →ₗ[R] I.map (algebraMap R S) :=
```

Comment:
The linter will ask for a documentation here.

Reply:
That got lost in copy pasting. Thanks!